### PR TITLE
The tracker!

### PR DESCRIPTION
**Pull request by Pamplemousse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+npm run watch
+```
+
+[Watchify](https://github.com/browserify/watchify) takes care of the rebuild on file changes and generates the `bundle.js` to be injected into webpages.
+
+
+## Resources
+
+  * [full list of DOM Events](https://developer.mozilla.org/en-US/docs/Web/Events)
+  * [webcompat/tinker-tester-developer-spy](https://github.com/webcompat/tinker-tester-developer-spy): some hooks have been copy/pasted in the [hooks folder](https://github.com/zaproxy/front-end-tracker/tree/master/hooks).
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: @psiinon , do you know what do I need to do to comply with [tinker-tester-developer-spy's license](https://github.com/webcompat/tinker-tester-developer-spy/blob/master/LICENSE)?

[the remark this answers is not in the document]
Comment (psiinon) [reply README.md]: Mozilla licence is good - its very permissive, we use it for Zest. So there will be no problem for our usage

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
-# front-end-tracker
\ No newline at end of file
+# Front-End Tracker
+
+![](https://api.travis-ci.com/Pamplemousse/front-end-tracker.svg?branch=master)
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+const topic = 'dom-events';
+mailbox.subscribe(topic, function(_, data) {
+  console.log(data);
+});
+```
+
+
+## Use with ZAP
+
+#### inject in all responses
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: README.md [hunk 4]
```diff
@@ -1 +1,64 @@
...
+
+[inject.js](https://github.com/zaproxy/community-scripts/blob/master/httpsender/inject_js_in_html_page.js) is an HttpSender script (see more [here](https://github.com/zaproxy/community-scripts/tree/master/httpsender)) to inject any kind of Javascript content into webpages that interest us.
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: README.md [hunk 5]
```diff
@@ -1 +1,64 @@
...
+
+Requirements:
+
+  * install the `script console` from the add-on marketplace
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: README.md [hunk 6]
```diff
@@ -1 +1,64 @@
...
+```
+npm run watch
+```
+
+[Watchify](https://github.com/browserify/watchify) takes care of the rebuild on file changes and generates the `bundle.js` to be injected into webpages.
+
+
+## Resources
+
+  * [full list of DOM Events](https://developer.mozilla.org/en-US/docs/Web/Events)
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: README.md [hunk 7]
```diff
@@ -1 +1,64 @@
...
+## Use with ZAP
+
+#### inject in all responses
+
+[inject.js](https://github.com/zaproxy/community-scripts/blob/master/httpsender/inject_js_in_html_page.js) is an HttpSender script (see more [here](https://github.com/zaproxy/community-scripts/tree/master/httpsender)) to inject any kind of Javascript content into webpages that interest us.
+
+Requirements:
+
+  * install the `script console` from the add-on marketplace
+  * install the `community scripts` from the add-on marketplace
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply README.md]: Done.

File: package.json [hunk 8]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@pamplemousse/front-end-tracker",
```
[the remark this answers is not in the document]
Comment (Pamplemousse) [reply package.json]: Done.

File: package-lock.json [hunk 9]
```diff
@@ -0,0 +1,6076 @@
+{
+  "name": "@pamplemousse/front-end-tracker",
```
Comment (thc202) [inline package-lock.json]: Needs update.

Comment (Pamplemousse) [reply package-lock.json]: Done.

**Pull request merged**